Sources/TuistDependenciesTesting/Carthage/Models/CarthageVersionFile+TestData.swift

```diff
@@ -225,6 +225,24 @@ extension CarthageVersionFile {
             tvOS: nil
         )
     }
+    
+    static var testAhoyRTC: Self {
```

Comment:
Nitpick: I'd not name the variable after the package. What about naming it `testWithoutContainer`?

Reply:
I just saw that other tests were named after packages and followed that) I also need to check that potential Graph issue. And I will update the changelog shortly, thanks!

Reply:
That was my convention for these test dates. I want to use "real" datas in unit tests. We can change name to more "case-specific". 

If we can stay with it I'd ask for small comment message to `testAhoyRTCJson ` you added that will be similar to: 

<img width="1014" alt="Zrzut ekranu 2021-07-8 o 17 39 10" src="https://user-images.githubusercontent.com/4774319/124951195-69440d00-e013-11eb-919a-9091a3c1752c.png">